boa3_test/tests/examples_tests/test_wrapped_neo.py

```diff
@@ -1,489 +1,596 @@
+        for x in range(len(invokes)):
+            self.assertEqual(expected_results[x], invokes[x].result)
 
-    def test_wrapped_neo_verify(self):
-        path = self.get_contract_path('wrapped_neo.py')
-        engine = TestEngine()
-        engine.use_contract_custom_name = self._use_custom_name
+        # transferring to someone other than yourself does change the balance
+        self.assertEqual(balance_sender_before.result - transferred_amount, balance_sender_after.result)
+        self.assertEqual(balance_receiver_before.result + transferred_amount, balance_receiver_after.result)
 
-        # should fail without signature
-        result = self.run_smart_contract(engine, path, 'verify')
-        self.assertEqual(False, result)
+        # should fail when any of the scripts' length is not 20
+        runner.call_contract(path, 'transfer', owner_script_hash, bytes(10), transferred_amount, "")
+        runner.execute()
+        self.assertEqual(VMState.FAULT, runner.vm_state, msg=runner.cli_log)
+        self.assertRegex(runner.error, self.ASSERT_RESULTED_FALSE_MSG)
 
-        # should fail if not signed by the owner
-        result = self.run_smart_contract(engine, path, 'verify',
-                                         signer_accounts=[self.OTHER_ACCOUNT_1])
-        self.assertEqual(False, result)
+        runner.call_contract(path, 'transfer', bytes(10), owner_script_hash, transferred_amount, "")
+        runner.execute()
+        self.assertEqual(VMState.FAULT, runner.vm_state, msg=runner.cli_log)
+        self.assertRegex(runner.error, self.ASSERT_RESULTED_FALSE_MSG)
 
-        result = self.run_smart_contract(engine, path, 'verify',
-                                         signer_accounts=[self.OWNER_SCRIPT_HASH])
-        self.assertEqual(True, result)
+        # should fail when the amount is less than 0
+        runner.call_contract(path, 'transfer', test_account_1_script_hash, owner_script_hash, -10, "")
+        runner.execute()
+        self.assertEqual(VMState.FAULT, runner.vm_state, msg=runner.cli_log)
+        self.assertRegex(runner.error, self.ASSERT_RESULTED_FALSE_MSG)
 
     def test_wrapped_neo_burn(self):
-        path = self.get_contract_path('wrapped_neo.py')
-        engine = TestEngine()
-        engine.use_contract_custom_name = self._use_custom_name
+        path, _ = self.get_deploy_file_paths('wrapped_neo.py')
+        runner = NeoTestRunner(runner_id=self.method_name())
 
-        self.run_smart_contract(engine, path, 'symbol')
-        wrapped_neo_address = engine.executed_script_hash.to_array()
+        runner.add_gas(self.OWNER.address, self.GAS_TO_DEPLOY)
+        wrapped_neo_contract = runner.deploy_contract(path, account=self.OWNER)
+        runner.update_contracts(export_checkpoint=True)
+        wrapped_neo_address = wrapped_neo_contract.script_hash
 
-        engine.reset_engine()
-        engine.add_neo(wrapped_neo_address, 10_000_000)
+        runner.add_neo(wrapped_neo_address, 10_000_000)
```

Comment:
same here, if possible avoid magic numbers
```suggestion
        initial_neo_token_supply = 10_000_000  # is this supposed to be 1 NEO? NEO doesn't have decimals anymore
        runner.add_neo(wrapped_neo_address, initial_neo_token_supply)
```